themes/SuiteP/css/tabs.scss

```diff
@@ -246,4 +246,9 @@
   border-top-right-radius: 4px;
   border-bottom-left-radius: 4px;
   border-bottom-right-radius: 4px;
-}
\ No newline at end of file
+}
+
+
+.tab-content .tab-pane-NOBOOTSTRAPTOGGLER:not(.active) {
```

Comment:
Explained that this white space is used for the mobile view, but the correct display:none was not in the desktop.